linalg/hypre.cpp

```diff
@@ -3253,12 +3253,18 @@ void HypreSmoother::SetOperator(const Operator &op)
    }
    if (l1_norms && pos_l1_norms)
    {
-#ifdef HYPRE_USING_CUDA
+#if defined(HYPRE_USING_CUDA)
       double *d_l1_norms = l1_norms;  // avoid *this capture
       CuWrap1D(height, [=] MFEM_DEVICE (int i)
       {
          d_l1_norms[i] = std::abs(d_l1_norms[i]);
       });
+#elif defined(HYPRE_USING_HIP)
+      double *d_l1_norms = l1_norms;  // avoid *this capture
+      HipWrap1D(height, [=] MFEM_DEVICE (int i)
```

Comment:
We should probably define a macro `GpuWrap1D` -- it could be defined just in this file.